### PR TITLE
Add docs page for migrating from Primer CSS

### PR DESCRIPTION
**Pull request by srt32**
closes https://github.com/github/design-systems/issues/1352

This change adds a new docs page to help consumers migrate from Primer
CSS to ViewComponents.

image

### CODE REVIEW

File: docs/content/migration.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+Many Primer CSS use cases are supported by Primer ViewComponents.
+
+When migrating from Primer CSS classes to ViewComponents, use this mapping to
+help guide your implementation. This list includes components currently in
+stable status.
+
+| Primer CSS Class | ViewComponent |
+|------------------|---------------|
+| `State`                   | `Primer::StateComponent`              |
```
Comment (joelhawksley) [inline docs/content/migration.md:13]: What do you think about having both of these columns be links to the respective libraries' page for each pattern?

Comment (srt32) [reply docs/content/migration.md:13]: Yup, can do! The bigger issue is some of these components are not actually in stable at the moment...

File: docs/content/migration.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+help guide your implementation. This list includes components currently in
+stable status.
+
+| Primer CSS Class | ViewComponent |
+|------------------|---------------|
+| [`State`](https://primer.style/css/components/labels#states)             | [`Primer::StateComponent`](https://primer.style/view-components/components/state)              |
+| [`breadcrumb-item`](https://primer.style/css/components/breadcrumb)      | [`Primer::BreadcrumbComponent`](https://primer.style/view-components/components/breadcrumb)    |
+| [`Counter`](https://primer.style/css/stickersheet/labels#counters)       | [`Primer::CounterComponent`](https://primer.style/view-components/components/counter)          |
+| [`Subhead`](https://primer.style/css/components/subhead)                 | [`Primer::SubheadComponent`](https://primer.style/view-components/components/subhead)          |
+| [`blankslate`](https://primer.style/css/components/blankslate)           | [`Primer::BlankslateComponent`](https://primer.style/view-components/components/blankslate)    |
```
Comment (manuelpuyol) [inline docs/content/migration.md:17]: could we auto-generate this somehow? 🤔

Comment (srt32) [reply docs/content/migration.md:17]: Probably but I'd love to start with something manual and then try that ;)

**Review by manuelpuyol — APPROVED** [review]

**Pull request merged**